### PR TITLE
fix(argo-rollouts): Added missing permissions for argo rollouts dashboard cluster role

### PR DESCRIPTION
**Pull request by talitz**
Following up this Issue: https://github.com/argoproj/argo-rollouts/issues/2738

On ArgoRollouts chart version 2.23.0 running on k8s cluster v1.22.17-eks, It seems like the follwoing permissions are missing:

`jfrog-argo-rollouts-dashboard" cannot get resource "leases" in API group "coordination.k8s.io" in the namespace `

After this commit it seems like the issue is resolved:

`time="2023-04-22T19:53:40Z" level=info msg="New leader elected: argo-rollouts-67fdd84bf6-q4ppv_c86f6b69-7aba-409d-a11c-1648edf1d8c3`

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

File: charts/argo-rollouts/Chart.yaml [hunk 1]
```diff
@@ -17,3 +17,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: added
       description: Ability to provide additional volumes and volumeMounts
+    - kind: added
+      description: Missing permissions to leases under coordination api group
```
Comment (yu-croco) [inline charts/argo-rollouts/Chart.yaml:21]: Please remove old changelog and make newline at the end of file.

File: charts/argo-rollouts/templates/dashboard/clusterrole.yaml [hunk 2]
```diff
@@ -71,4 +71,12 @@ rules:
...
-{{- end }}
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - create
+      - get
+      - update
+{{- end }}
```
Comment (yu-croco) [inline charts/argo-rollouts/templates/dashboard/clusterrole.yaml:82]: Please make newline at the end of file.

File: charts/argo-rollouts/templates/dashboard/clusterrole.yaml [hunk 3]
```diff
@@ -71,4 +71,12 @@ rules:
     verbs:
       - list
       - watch
-{{- end }}
+  - apiGroups:
```
Comment (yu-croco) [inline charts/argo-rollouts/templates/dashboard/clusterrole.yaml:74]: 📝 Policy is from [here](https://github.com/argoproj/argo-rollouts/blob/master/manifests/role/argo-rollouts-clusterrole.yaml#L83-L91).
LGTM.

image

**Review by yu-croco — COMMENTED** [review]
Hi @talitz , thank you for your PR.
I left some comments, otherwise LGTM.

**Review by jmeridth — APPROVED** [review]

**Comment by talitz** [comment]
Sorry, I made some messes with the commits here. Closing this PR and please let's follow up here: https://github.com/argoproj/argo-helm/pull/1988.

**Pull request closed without merging**